docker-compose.x86.darknet.gpu.yml

```diff
@@ -0,0 +1,21 @@
+version: '2.4'
+
+services:
+  ml_api:
+    image: thespaghettydetective/ml_api_x86_64_darknet_gpu
```

Comment:
Do you think it's possible that we use the same image for both gpu and cpu? So basically if the GPU hardware is present, the user only need to set "HAS_GPU" to "True", without having to change the image and how the start the server. This will make documentation and supporting a lot easier.

Reply:
Image size is a good point. The time to launch a new Obico server is already pretty long now. I'll think a little more about options here.